app/helpers/injection_helper.rb

```diff
@@ -69,7 +69,12 @@ def inject_current_hub
   end
 
   def inject_current_order
-    inject_json_ams "currentOrder", current_order, Api::CurrentOrderSerializer, current_distributor: current_distributor, current_order_cycle: current_order_cycle
+    inject_json_ams "currentOrder",
+                    current_order,
+                    Api::CurrentOrderSerializer,
+                    current_distributor: current_distributor,
+                    current_order_cycle: current_order_cycle,
+                    line_items: current_order_line_items
```

Comment:
I think this needs a unit test in injection_helper_spec or just extend the existing test to validate order.line_items are correct.